src/Schema/Events/Toggle.ts

```diff
@@ -35,3 +35,31 @@ export interface ToggledNotification {
   original: number
   subject: PushNotificationType
 }
+
+/**
+ * A user toggles a saved search on/off on iOS
+ *
+ *  This schema describes events sent to Segment from [[toggledNotification]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "toggledSavedSearch",
+ *    context_screen_owner_type: "artist",
+ *    context_screen_owner_id: "58de681f275b2464fcdde097",
+ *    context_screen_owner_slug: "anthony-hunter",
+ *    modified: 1,
+ *    original: 0,
+ *    saved_search_id: "58de681f275b2464fcdde097"
+ *  }
+ * ```
+ */
+export interface ToggledSavedSearch {
```

Comment:
Do we usually model a single `Toggle` event to capture two different states or would distinct `EnabledSavedSearch` and `DisabledSavedSearch` be useful?

Reply:
I think toggle is ok! We can use the modified/original properties to figure out what happened!